instagram/src/components/Login/Login.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Login extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            username: '',
+            password: ''
+        }
+    }
+
+    // Update Username
+
+    updateUsername = event => {
+        this.setState({
+            username: event.target.value
+        })
+    }
+
+    // Update Password
+
+    updatePassword = event => {
+        this.setState({
+            password: event.target.value
+        })
+    }
+
+    // Login Function
+
```

Comment:
You're trying to update user's credential here, but you are comparing them to hard coded values of `fozcat` and `test` in withAuthenticate.js line 32, which will return false unless you're using the hardcoded values to login.